scripts/boston_cityhall_elec.json

```diff
@@ -0,0 +1,36 @@
+{
+    "citation": "Department of Innovation and Technology",
+    "description": "Electric power load at City Hall (1 City Hall Square) measured every 15 minutes. And it is continously updated.",
+    "homepage": "https://data.boston.gov/dataset/city-hall-electricity-usage",
+    "keywords": ["City Hall", "Demand", "Electricity", "Energy", "Load", "Meter", "Power"],
+    "licenses": [
+        {
+            "name": "Open Data Commons Public Domain Dedication and License (PDDL)"
+        }
+    ],
+    "name": "boston-cityhall-electricity-usage",
```

Comment:
If you choose name `boston-cityhall-electricity-usage`
then the file should be named as `boston_cityhall_electricity_usage.json`

Reply:
I have corrected the name of the file.

Reply:
Note the _ vs -